lib/evm_database.rb

```diff
@@ -116,6 +116,11 @@ def self.write_expected_schema(connection = ActiveRecord::Base.connection)
     File.write(SCHEMA_FILE, current_schema(connection).to_yaml)
   end
 
+  def self.raise_failover_executed_event
+    msg = "Server_IP: #{MiqServer.my_server.ipaddress}, Server Host Name: #{MiqServer.my_server.hostname}"
```

Comment:
This is odd, why is this "Server_IP" why not "Server IP address" or something more human readable?